plumbing/mpool/pool.go

```diff
@@ -0,0 +1,49 @@
+package mpool
+
+import (
+	"context"
+
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	"gx/ipfs/QmVRxA4J3UPQpw74dLrQ6NJkfysCA1H4GU28gVpXQt9zMU/go-libp2p-pubsub"
+
+	"github.com/filecoin-project/go-filecoin/core"
+	"github.com/filecoin-project/go-filecoin/plumbing/msg"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+// Pool interfaces to the core message pool
+type Pool struct {
+	pool   *core.MessagePool
+	pubsub *pubsub.PubSub
+}
+
+// New builds a new Pool.
+func New(pool *core.MessagePool, pubsub *pubsub.PubSub) *Pool {
+	return &Pool{pool: pool, pubsub: pubsub}
+}
+
+// Pending lists un-mined messages in the pool
+func (ls *Pool) Pending(ctx context.Context, messageCount uint) ([]*types.SignedMessage, error) {
```

Comment:
should probably say what messagecount does and is for, waits for that many messages and its used for testing. 

actually this is a pretty weird thing to have in the interface. if we didn't have to support it we could just have the core.messagepool in the plumbing and we wouldn't need this wrapper. if pubsub was available in plumbing then a porcelain call could implement this wait functionality by depending on the plumbing pending() and messagepubsub() calls.

Reply:
Done!